features/step_definitions/clear_data_on_signout_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given(/^I am on the Money Manager tool$/) do
+  visit('/en/tools/money-manager/')
+end
+
+Then(/^I complete the series of questions$/) do
+  find_field(id: 'questionnaire_received_first_payment_true').click
```

Comment:
We're refactoring this now, using the other specs as a guide. It looks like the other specs are using some kind of helper (e.g. `debt_management_page`) instead of instantiating the SitePrism page directly (e.g. `DebtManagement.new`, which is what the SitePrism readme suggests), but we can't see where these are coming from. We assumed they're generated automatically by some metaprogramming somewhere, but we don't have a `money_manager_page` available after creating a MoneyManager class. Could you point us towards any code or docs that will help us understand this?